src/main/java/com/paneedah/mwc/items/guns/MAC10Factory.java

```diff
@@ -21,721 +16,591 @@
 import net.minecraft.util.math.Vec3d;
 import org.lwjgl.opengl.GL11;
 
-import java.util.Arrays;
-
 public class MAC10Factory {
 
     public Item createGun(CommonProxy commonProxy) {
         return new Weapon.Builder()
+                .withName("mac10")
+                .withFireRate(0.999f)
+                .withRecoil(2.5f)
+                .withZoom(0.9f)
+                .withConfigGroup(GunConfigurationGroup.SUBMACHINE_GUN)
+                .withShellType(Type.PISTOL)
+                .withMuzzlePosition(new Vec3d(-.16, -0.95, -2.8))
+                .withMaxShots(1, Integer.MAX_VALUE)
+                .withShootSound("mac10")
+                .withSilencedShootSound("mp5_silenced")
+                .withReloadSound("mac10_reload")
+                .withUnloadSound("mac10_unload")
+                .withInspectSound("inspection")
+                .withDrawSound("noaction_draw")
+                .withReloadingTime(45)
+                .withFlashIntensity(0.5f)
+                .withFlashScale(() -> 0.6f)
+                .withFlashOffsetX(() -> 0.16f)
+                .withFlashOffsetY(() -> 0.16f)
+                .withInaccuracy(2f)
```

Comment:
No balance changes until 0.2.
```suggestion
                .withInaccuracy(3f)
```